src/frontend/next/src/components/SearchInput/AuthorSearchInput.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+interface SearchInput {
+  text: string;
+  onChange: () => void;
+  classes: {
+    input: string;
+  };
+}
+
+function AuthorSearchInput({ text, onChange, classes }: SearchInput) {
```

Comment:
Take a look at  https://github.com/Seneca-CDOT/telescope/pull/1485/files#diff-dfc72a93d99668af007bd9cfe8dbb5403cefb490f108b9ead155075124b4ef68R5-R7 to see how this part should be done

Reply:
Thanks for the instruction. I will update it